url/strip.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// This module is browser compatible.
+
+/**
+ * Strips any hash (eg. `#header`) or search parameters (eg. `?foo=bar`) from the provided URL.
+ *
+ * (Mutates the original url provided)
+ * @param url url to be stripped.
+ */
+export function strip(url: URL) {
```

Comment:
I think this is too trivial to expose as a public API (`url.hash = ""; url.search = "";` is enough easy to perform)

Reply:
Done.